x/evm/types/types.go

```diff
@@ -72,15 +72,15 @@ const (
 		}
 	]`
 	AxelarGatewayCommandMintToken            = "mintToken"
-	mintTokenMaxGasCost                      = 200000
+	mintTokenMaxGasCost                      = 150000
```

Comment:
Mints seem to only cost `80000` rn. I would imagine it should be cheaper with the optimizations? Should we keep it more around `120k`?

I guess in the worst case, we can still retry a batch with a higher gas limit?

Reply:
mint isn't that cheap. we didn't another round of estimate already

Reply:
My point was that it seemed like we optimized gas costs quite a bit, so if "rn" (mainnet version) it's `80000`, I'd imagine it'll be even lower in the latest version, so just surprised by that.

Reply:
Optimizations were applied to the new version which carries a lot of new features. It's never expected to be better than the previous version but better than a non-optimized latest version.